extensions/mssql/src/objectExplorerNodeProvider/fileSources.ts

```diff
@@ -187,18 +188,23 @@ export class HdfsFileSource implements IFileSource {
 				reject(error);
 			});
 
+			let data: any[] = [];
 			if (maxBytes) {
 				remoteFileStream = remoteFileStream.pipe(meter(maxBytes));
 				remoteFileStream.on('error', (err) => {
 					error = <HdfsError>err;
 					if (error.message.includes('Stream exceeded specified max')) {
-						error.message = localize('maxSizeReached', 'File exceeds max size of ${0}', bytes(maxBytes));
+						// We have data > maxbytes, show we're truncating
+						let previewNote: string = '########################### NOTICE: This file has been truncated at '+ bytes(maxBytes) +' for preview ########################### \r\n';
+						data.splice(0, 0, Buffer.from(previewNote, 'utf-8'));
+						vscode.window.showWarningMessage(localize('maxSizeReached', "Preview only loads {0} data of the file and truncates the rest.", bytes(maxBytes)));
```

Comment:
Please change the message. Might want to check with PM, but I would suggest
```suggestion
						vscode.window.showWarningMessage(localize('maxSizeReached', "The previewed file {0} has been truncated - only the first {1} byes are shown.",  remotePath, bytes(maxBytes)));
```

Reply:
Checked with Rony, he suggested to add the same message we're appending to the text file i.e. "The file has been truncated at 30 MB for preview." Made the changes.